### PR TITLE
Deploy showcase, support when building gh-pages on master.

### PR DESCRIPTION
**Pull request by hramos**
Addresses #9865.

We want the showcase and support pages to be loaded from root regardless of which React Native version is selected.

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @bestander and @Andreyco to be potential reviewers.

File: website/publish-gh-pages.js [hunk 1]
```diff
@@ -110,11 +110,15 @@ if (!CI_PULL_REQUEST && CIRCLE_PROJECT_USERNAME === `facebook`) {
       .map(file => `../react-native/${file}`);
```
Comment (bestander) [inline website/publish-gh-pages.js]: almost there, need to fix `blog` comparisons in a few places above

**Comment by facebook-github-bot-8** [comment]
@hramos updated the pull request - [view changes](https://github.com/facebook/react-native/pull/10004/files/3993a82afab6a6e3eafcf9b241a84bcf80b389eb..f7603b411d31032c6c3f9d5953faffc88176b841)

**Comment by bestander** [comment]
nice work!
Let's give it a try

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**